src/views/Drawer/DrawerSidebar.js

```diff
@@ -110,21 +113,20 @@ class DrawerSidebar extends React.PureComponent<Props> {
     const { state } = this.props.navigation;
     invariant(typeof state.index === 'number', 'should be set');
     return (
-      <View style={[styles.container, this.props.style]}>
```

Comment:
looks like we're not using styles.container or `this.props.style` anymore, intentional?

Reply:
Ah, that was a mistake. Originally replaced with SafeAreaView, then removed SafeAreaView. It should be put back in.